### DISCUSSION TITLE
Filament repeater

### DISCUSSION THREAD

**Discussion by webmclink** [filamentphp/filament#9135] [Help] [answered]
I have a form using the repeater, is there a way to prevent the user from selecting the same value? The candidate dropdown should be unique. Thank you

![image](https://github.com/filamentphp/filament/assets/60499148/76f463fb-5d2b-4a96-88fc-0e5955e20ed4)

**Comment by leandrocfe** [comment, marked as answer]
https://github.com/filamentphp/filament/discussions/8561#discussioncomment-7034671

**Reply by webmclink** [reply]
Thank you
